cypress/integration/statistic/statistic.test.js

```diff
@@ -0,0 +1,134 @@
+/// <reference types="cypress" />
+import { config } from '../../../src/configs';
+
+import {
+  getPopularCategories,
+  getPopularProducts,
+  getPopularCategoriesEmpty,
+  getPopularProductsEmpty,
+  getPaidOrdersStatistic,
+  getOrdersStatistic
+} from './statistic.variables';
+
+describe('Statistic page test', () => {
+  beforeEach(() => {
+    cy.login(Cypress.env('ADMIN_LOGIN'), Cypress.env('ADMIN_PASSWORD'));
+    cy.stubRequest('getPopularCategories', getPopularCategories).as(
+      'getPopularCategories'
+    );
+    cy.stubRequest('getPopularProducts', getPopularProducts).as(
+      'getPopularProducts'
+    );
+    cy.visit('/');
+    cy.wait(3000);
+
+    cy.get('[href="/stats"]').click();
```

Comment:
Its better to set here link from configs

Reply:
done